features/product/viewing_products/viewing_sorted_products.feature

```diff
@@ -20,16 +20,18 @@ Feature: Sorting listed products
         And this product belongs to "Fluffy Pets"
         And this product's price is "$13.27"
 
-    @ui
+    @api @ui
     Scenario: Sorting products by their dates with descending order
-        When I view newest products from taxon "Fluffy Pets"
+        When I browse products from taxon "Fluffy Pets"
+        And I sort products by the newest date first
```

Comment:
It could have stayed as one step

Reply:
I wanted to be more consistent with the rest of the tests. The second case is that, in fact, from the UI perspective we cannot open a specific taxon and have already sorted the products by anything other than position

Reply:
Yes, you are right and I am OK with that, but from the business perspective nothing has changed and the user's intention is to view sorted products regardless of the number of steps. So still, it could have stayed as one step, but it is OK as it is right now 😃

Reply:
Okay, now I see what you meant :)